.github/workflows/docs.yml

```
@@ -126,8 +126,8 @@ jobs:
           # Wait for server to start
           sleep 5
           
-          # Test that server is running
-          curl -f http://localhost:8000/ || exit 1
+          # Test that server is running (MkDocs serves with /safe-job/ path)
+          curl -f http://localhost:8000/safe-job/ || exit 1
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)